d3a_interface/utils.py

```diff
@@ -222,8 +222,9 @@ def format_datetime(datetime, ui_format=False, unix_time=False):
         return datetime.format(DATE_TIME_FORMAT)
 
 
-def datetime_to_string_incl_seconds(datetime):
-    return datetime.format(DATE_TIME_FORMAT_SECONDS)
+def datetime_to_string_incl_seconds(date_time: Optional[DateTime]) -> Optional[str]:
+    if date_time:
+        return date_time.format(DATE_TIME_FORMAT_SECONDS)
```

Comment:
Why is the `date_time` set as optional? AFAIS it doesn't accept a default value

Reply:
Because it could be None.

Reply:
Nit: I'm not completely sure about this solution: why should this method accept `None`, when this is the only operation it performs? What I mean is: if the value is `None`, why are we calling the method?

Reply:
I think this is remnant of testing. I will remove the None check.